library/sql.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
@Otis0620 Wouldn't it be appropriate to UP_CASE these constant names? Or is that not JavaScript-ish?

Reply:
@ybakos yah javascript is weird that way. If it is an import or destructuring then they usually stay lowercase, but if it is something like const NUM_DAYS = 11; (upper snake case) then it is uppercase.